tmt/base.py

```diff
@@ -2046,7 +2046,7 @@ class Result(object):
         'error': 'magenta',
         }
 
-    def __init__(self, data, name):
+    def __init__(self, data, name, interpret=None):
```

Comment:
IMHO should be 'respect' as per https://tmt.readthedocs.io/en/stable/spec/tests.html#result
Btw - This is making unitest fail.

Reply:
I guess so. Let's have it confirmed by @psss.

Reply:
Yeah, makes sense, the default value should be `respect`.